### PR TITLE
Refactor/convenience frames

### PR DESCRIPTION
**Pull request by TanjaSmiley**
## Proposed changes

The approach frame is now used for the kicking and the ball_kick_area decision. After the changes for usage of the localization in the behavior are done, we can evaluate if we want to use other convenience frame as well or the approach frame for other actions or decisions. To not interfere with the other decisions and actions a new method in the world model capsule returning the ball position in the approach frame has been created.
## Related issues

#68 
## Necessary checks
- [x] Update package version
- [x] Run `catkin build`
- [ ] Write documentation
- [ ] Create issues for future work
- [x] Test on your machine
- [ ] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

File: bitbots_blackboard/src/bitbots_blackboard/capsules/world_model_capsule.py [hunk 1]
```diff
@@ -58,6 +58,13 @@ def get_ball_stamped(self):
...
 
+    def get_ball_position_uv_approach_frame(self):
+        try:
+            ball_position = self.tf_buffer.transform(self.ball, 'approach_frame', timeout=rospy.Duration(0.3))
+        except (tf2.ConnectivityException, tf2.LookupException, tf2.ExtrapolationException) as e:
+            rospy.logwarn(e)
+        return ball_position.point.x, ball_position.point.y
+
     def get_ball_distance(self):
         u, v = self.get_ball_position_uv()
```
Comment (NFiedler) [inline bitbots_blackboard/src/bitbots_blackboard/capsules/world_model_capsule.py]: we should use the approach frame for the ball distance. Possibly the distance to everything in the field.

**Comment by NFiedler** [comment]
I wonder whether it would be better to use the approach frame generally for all move base commands. the pose in the approach frame might be more accurate.

**Comment by Flova** [comment]
I tried to test these changes but the branch is quite a bit behind the master now.

**Review by NFiedler — DISMISSED** [review]
LGTM 
Works for me.
I was involved in this so somebody else (@timonegk ?) review this please.

**Review by NFiedler — DISMISSED** [review]

**Review by timonegk — DISMISSED** [review]

**Review by NFiedler — APPROVED** [review]

**Pull request merged**